lib/main-app.js

```diff
@@ -128,7 +147,24 @@ app.on('ready', function () {
   checkUpdate()
   ipcServer = require('./ipcServer')
   ipcServer.server.start()
+  
+  mainWindow.webContents.on('did-finish-load', function () {
+    //if app was launched with open-url
+    if (shouldOpenUrl) {
+      openURL(urlToOpen);
+      shouldOpenUrl = false;
+    }
+  });
 })
 
+//Listen to custom protocol incoming messages (if app not started, called before ready event)
+app.on('open-url', function (event, url) {
+  event.preventDefault();
+  //open URL if app ready - if not it means app was closed when URL was clicked. URL will be opened once app has fully loaded (webContents did-finish-load).
+  if (app.isReady()) openURL(url);
+  else { shouldOpenUrl = true; urlToOpen = url; }
```

Comment:
Please do not use semi colon. We compliant [JavaScript Standard Style](https://github.com/feross/standard) :bow: